packaging/push_chart.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -e
+
+# This will be running in automation so we don't want the
+# interactive pager to pop up
+export AWS_PAGER=""
+# The circle job ensures helm is installed, but if for some reason
+# someone is running this locally, then alert them if helm isn't
+# installed.
+if ! command -v helm 2> /dev/null; then
+    echo "Helm isn't installed. Instructions at https://helm.sh/docs/intro/install/"
+    exit 1
+fi
+
+# Get the toplevel dir of the repo so we can run this command
+# no matter which directory we are in.
+TOP_DIR="$( git rev-parse --show-toplevel)"
+echo $TOP_DIR
+
+CHART_VERSION=$(grep version: $TOP_DIR/charts/telepresence/Chart.yaml | awk ' { print $2 }')
+PACKAGE_FILE=telepresence-$CHART_VERSION.tgz
+CHART_ARTIFACT_DIR=$(mktemp -d)
+echo "Artifacts will be stored in ${CHART_ARTIFACT_DIR}"
+helm package $TOP_DIR/charts/telepresence -d $CHART_ARTIFACT_DIR
+
+CHART_PACKAGE=$CHART_ARTIFACT_DIR/$PACKAGE_FILE
+echo "CHART_PACKAGE is here: ${CHART_PACKAGE}"
+
+bucket_dir=
+if [[ -n "${BUCKET_DIR}" ]]; then
+    bucket_dir="${BUCKET_DIR}"
+else
+    bucket_dir="charts-dev"
+fi
+
+if [ -z "$AWS_BUCKET"] ; then
```

Comment:
You definitely need a space between the `"` and the `]`.

On that note, running `shellcheck` on our shell scripts from `make lint` would be a good enhancement.